packages/trader/src/AppV2/Containers/Trade/trade.tsx

```diff
@@ -131,7 +148,10 @@ const Trade = observer(() => {
             )}
             <ServiceErrorSheet />
             <ClosedMarketMessage />
-            <TradeErrorSnackbar error_fields={['stop_loss', 'take_profit', 'date_start']} should_show_snackbar />
+            <TradeErrorSnackbar
```

Comment:
Removed hardcoded snackbar for Stake, but added stake errors to the array with observables

Reply:
are these all the errors that exist?

i was working on another card and noticed we have 9 types of trade parameters